openmp/libomptarget/plugins-nextgen/common/PluginInterface/PluginInterface.cpp

```diff
@@ -274,7 +317,7 @@ struct RecordReplayTy {
   void saveKernelOutputInfo(const char *Name) {
     SmallString<128> OutputFilename = {
         Name, (isRecording() ? ".original.output" : ".replay.output")};
```

Comment:
Create a new extensions when outputting in the diff format, e.g., `.replay.diff.output` to avoid mixing up verification from the `llvm-omp-kernel-replay` tool

Reply:
The motivation was to save the disk usage from the memory dumps. So having the diff dump along with entire dump defeats the purpose. The verification compares the `.original.output` to the `.replay.output` anyway. Since both are now replaced by diffs, the verification from the `llvm-omp-kernel-replay` tool just compares the diffs now.